tests/unit/mixins/esri-module-cache-test.js

```diff
@@ -1,10 +1,17 @@
+/* eslint-disable ember/no-new-mixins */
+/* eslint-disable ember/no-mixins */
 import EmberObject from '@ember/object';
 import EsriModuleCacheMixin from 'ember-esri-loader/mixins/esri-module-cache';
-import { module } from 'qunit';
-import test from 'ember-sinon-qunit/test-support/test';
+import { module, test } from 'qunit';
+import sinon from 'sinon';
```

Comment:
I started getting errors along the way when tests used sinon stuff.  It looks like some of the changes to `qunit`/`ember-qunit` that the updates pulled in broke this.  I tried updating to the latest `ember-sinon-qunit`, but that didn't fix it.  Just using sinon directly seemed easy enough, though it is now our responsibility to explicitly call `sinon.restore()` after tests.